Sources/MapboxSearch/PublicAPI/SearchOptions.swift

```diff
@@ -285,7 +285,8 @@ public struct SearchOptions {
             offlineSearchPlacesOutsideBbox: offlineSearchPlacesOutsideBbox,
             ensureResultsPerCategory: nil,
             // TODO: NAVIOS-2054 Support multiple categories search and ability to ensure results per category.
-            attributeSets: attributeSets?.map { NSNumber(value: $0.coreValue.rawValue) }
+            attributeSets: attributeSets?.map { NSNumber(value: $0.coreValue.rawValue) }, 
+            evSearchOptions: nil
```

Comment:
Could you please also create a ticket to support this if it's not already created? and add a TODO, similar as for ensureResultsPerCategory